cmd/kpod/save.go

```diff
@@ -12,11 +12,13 @@ import (
 
 const (
 	dockerArchive = "docker-archive:"
+	ociArchive    = "oci-archive:"
```

Comment:
Can we move these constants into Libkpod? Other users of libkpod might want to use them

Reply:
Maybe remove the ":" and append it back when making the URI to save to

Reply:
fixed

Reply:
Ideally I’d prefer code not to paste/concatenate the strings at all, and call `containers/image/docker/archive.ParseReference(input)` and `containers/image/oci/archive.NewReference(path, image)` directly. [Perhaps there should be a `docker/archive.NewReference` as well.]

I realize that strings are convenient, and explicit types and stricter discipline about using them is a cost which might or might not pay off; consider at least something like
```go
// import the two c/image subpackages
var (
   dockerArchivePrefix = dockerarchive.Transport.Name() + ":"
   ociArchivePrefix = ociarchive.Transport.Name() + ":"
)
```